src/panels/config/integrations/integration-panels/thread/thread-config-panel.ts

```diff
@@ -635,6 +635,9 @@ export class ThreadConfigPanel extends SubscribeMixin(LitElement) {
         cursor: default;
         overflow: visible;
       }
+      ha-list-item img {
+        border-radius: 0;
+      }
```

Comment:
Or set `graphic` to `medium` and
```suggestion
      ha-list-item {
        --mdc-list-item-graphic-size: 40px;
      }
```
?

Reply:
There is an issue when trying to set the size of the item.

```css
--mdc-list-item-graphic-size: 40px;
```

![CleanShot 2023-10-05 at 14 35 17](https://github.com/home-assistant/frontend/assets/5878303/14cfb609-c6db-44c3-97b0-1b4e19e578d1)

So we need to do this :
```css
--mdc-list-item-graphic-size: 40px;
height: 40px;
```

Setting border radius was easier 😅

Reply:
stupid thing...